addons/web_editor/static/src/js/wysiwyg/dialog.js

```diff
@@ -36,7 +36,14 @@ var WysiwygDialog = Dialog.extend({
 
         var self = this;
         this.opened(function () {
-            self.$('input:visible:first').focus();
+            // Issue seems to be, at least in part, calling .focus() too early
+
+            // self.$('input:visible:first')[0].addEventListener('load', function () { 
+            //     self.$('input:visible:first').focus();
+            // }, false);
+            // Solution above (and similar attempts with .on()) does not trigger (too late ?)
+            setTimeout(function () { self.$('input:visible:first').focus(); }, 20);
+            // Timeout works but is hackish, may fail if loading of input is very slow (very unlikely)
```

Comment:
@dmo-odoo I appreciate your feedback as well, regarding the fix itself and whether it may be joined with task
https://www.odoo.com/web#id=3224551&model=project.task

Reply:
See task https://www.odoo.com/web#id=3138700&cids=1&model=project.task&view_type=form